.github/workflows/msbuild.yml

```diff
@@ -0,0 +1,44 @@
+name: MSBuild
+
+on: [push]
+
+env:
+  # Path to the solution file relative to the root of the project.
+  SOLUTION_FILE_PATH: .
+
+  # Configuration type to build.
+  # You can convert this to a build matrix if you need coverage of multiple configuration types.
+  # https://docs.github.com/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+  BUILD_CONFIGURATION: Debug
```

Comment:
The matrix seems really easy to do, worth a shot so there are both executables? :)

Reply:
Otherwise you can do it with msbuild "pseudo"-projects, that way you build them in parallel in the same msbuild instance

Reply:
I already wrote some matrixes, they are also quite easy to setup, i will use them to also give us Release builds.